packages/email-service/test/app.e2e-spec.ts

```diff
@@ -53,6 +54,50 @@ afterAll(() => {
   app.close();
 });
 
-it('Dummy test', () => {
-  expect('1').toEqual('1');
+it('should successfully register a sender', async () => {
+  const response: EmailProto.RegisterSenderResponse = await new Promise(
+    (resolve, reject) => {
+      emailClient.registerSender(
+        {
+          from_email: 'example@example.com',
+          from_name: 'example',
+          reply_to: 'example@example.com',
```

Comment:
Tests seem to be failing based on camel vs snake case, ts-proto converts to camel case afaik